External/Plugins/ResultsPanel/PluginMain.cs

```diff
@@ -122,13 +125,26 @@ public void HandleEvent(Object sender, NotifyEvent e, HandlingPriority priority)
                     if (evnt.Action == "ResultsPanel.ClearResults")
                     {
                         e.Handled = true;
-                        this.pluginUI.ClearOutput();
+
+                        if (evnt.Data == null)
```

Comment:
Could you make the deault results panel also like another panel inside the panel helper? Having this kind of differences makes code slightly more complex and harder to read.